lib/Mandate.php

```diff
@@ -7,12 +7,12 @@
  *
  * @property string $id
  * @property string $object
- * @property mixed $customer_acceptance
+ * @property \Stripe\StripeObject $customer_acceptance
  * @property bool $livemode
- * @property mixed|null $multi_use
```

Comment:
source-style polymorphism

Reply:
yeah discussed with ob that this could be problematic because that supports `hide_nil` and so it technically can be undefined. But we should fix in a future PR